.github/workflows/build_test_deploy.yml

```
@@ -445,6 +445,36 @@ jobs:
       - run: '[[ -z "$BROWSERSTACK_ACCESS_KEY" ]] && echo "Skipping for PR" || node run-tests.js test/integration/production-nav/test/index.test.js'
         if: ${{needs.build.outputs.docsChange != 'docs only change'}}
 
+  testFirefoxNode17:
+    name: Test Firefox Node.js 17
+    runs-on: ubuntu-latest
+    needs: [build, testSafari, build-native-dev]
```

```suggestion
    needs: [build, testFirefox, build-native-dev]
```
Should this rely on the existing `testFirefox` job instead of `testSafari`?